Algorithms/Search/Linear search.cpp

```diff
@@ -0,0 +1,27 @@
+#include<bits/stdc++.h>
```

Comment:
The code is fine @kvanshika10 but please list linear search in documentation.md also.